app/views/pages/security.html.erb

```diff
@@ -0,0 +1,211 @@
+<% @title = "Security" %>
+
+<div class="l-colspan t-body">
+  <p>RubyGems.org Security</p>
+
+  <h2>Reporting a Rubygems or RubyGems.org security bug</h2>
+
+  <p>
+    For Rails, see <a href="http://rubyonrails.org/security/">Rails Security</a>
+  </p>
+
+  <h3>Getting a CVE</h3>
+
+  <p>
+    See the <a href="http://guides.rubygems.org/security/#reporting-security-vulnerabilities">
```

Comment:
This page is really only for vulns with rubygems.org itself. We will link to these.